mindsdb/integrations/handlers/apache_doris_handler/apache_doris_handler.py

```diff
@@ -0,0 +1,198 @@
+from collections import OrderedDict
+
+import pandas as pd
+import mysql.connector
+
+from mindsdb_sql import parse_sql
+from mindsdb_sql.parser.ast.base import ASTNode
+
+from mindsdb.utilities import log
+from mindsdb.integrations.libs.base import DatabaseHandler
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+    HandlerResponse as Response,
+    RESPONSE_TYPE
+)
+from mindsdb.integrations.libs.const import HANDLER_CONNECTION_ARG_TYPE as ARG_TYPE
+
+
+class ApacheDorisHandler(DatabaseHandler):
```

Comment:
Wouldn't be easy to extend MySQLHandler? Or there is some difference?

Reply:
I had the same concern but @martyna-mindsdb said that it needs its own handler so I made this. There is not much difference though.

Reply:
@aditya-azad 
You could have utilized the existing [MySQLHandler](https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/mysql_handler/mysql_handler.py#L21).